panel/src/main.scss

```diff
@@ -19,119 +20,149 @@ $layers: (
-$color-positive-on-dark: #a7bd68;
-$color-focus: #4271ae;
-$color-focus-border: $color-focus;
+$color-positive-on-dark: $color-green-400;
+$color-focus: $color-blue-600;
 $color-focus-outline: rgba($color-focus, 0.25);
-$color-focus-on-dark: #81a2be;
-$color-notice: #f5871f;
-$color-notice-on-dark: #de935f;
-$color-negative: #c82829;
-$color-negative-border: $color-negative;
+$color-focus-on-dark: $color-blue-400;
+$color-notice: $color-orange-600;
+$color-notice-on-dark: $color-orange-400;
+$color-negative: $color-red-600;
 $color-negative-outline: rgba($color-negative, 0.25);
-$color-negative-on-dark: #d16464;
-$color-border: #ccc;
-$color-backdrop: rgba($color-dark, 0.6);
-$color-inset: #ebebeb;
+$color-negative-on-dark: $color-red-400;
 
 /**
- * Typography
+ * Text sizes
  */
-$font-size-tiny: 0.75rem;
-$font-size-small: 0.875rem;
-$font-size-medium: 1rem;
-$font-size-large: 1.25rem;
-$font-size-huge: 1.5rem;
-$font-size-monster: 1.75rem;
-
-$font-family-sans: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
-$font-family-mono: "SFMono-Regular", Consolas, Liberation Mono, Menlo, Courier, monospace;
-
-$font-weight-normal: 400;
-$font-weight-bold: 600;
+$text-xs: 0.75rem;
+$text-sm: 0.875rem;
+$text-base: 1rem;
+$text-lg: 1.125rem;
+$text-xl: 1.25rem;
+$text-2xl: 1.5rem;
+$text-3xl: 1.875rem;
+$text-4xl: 2.25rem;
+$text-5xl: 3rem;
+$text-6xl: 4rem;
 
 /**
- * Fields
+ * Font families
  */
-$field-input-padding: .5rem;
-$field-input-height: 2.25rem;
-$field-input-line-height: 1.25rem;
-$field-input-font-size: $font-size-medium;
-$field-input-color-before: $color-dark-grey;
-$field-input-color-after: $color-dark-grey;
-$field-input-border: 1px solid $color-border;
-$field-input-focus-border: 1px solid $color-focus-border;
-$field-input-focus-outline: 2px solid $color-focus-outline;
-$field-input-invalid-border: 1px solid $color-negative-outline;
-$field-input-invalid-outline: 0;
-$field-input-invalid-focus-border: 1px solid $color-negative-border;
-$field-input-invalid-focus-outline: 2px solid $color-negative-outline;
-$field-input-background: $color-white;
-$field-input-disabled-color: $color-light-grey;
-$field-input-disabled-background: lighten($color-background, 2.5%);
-$field-input-disabled-border: 1px solid lighten($color-border, 2%);
+$font-sans: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+$font-mono: "SFMono-Regular", Consolas, Liberation Mono, Menlo, Courier, monospace;
 
 /**
- * Lists
+ * Font weights
  */
-$list-item-height: calc(2rem + 2px);
-$list-item-text-color: $color-dark;
-$list-item-text-font-size: $font-size-small;
-$list-item-info-color: $color-dark-grey;
-$list-item-info-font-size: $font-size-tiny;
+$font-normal: 400;
+$font-bold: 600;
 
 /**
- * Misc
+ * Border radius
  */
-$border-radius: 1px;
+$rounded-xs: 1px;
```

Comment:
Why this one is in `px` while other are `rem` ?

Reply:
@yoanmalie Cause 1px should stay 1px no matter what the root font size is. And it's also much easier o read than any weird `rem` fraction in this case.